docs/staff/editor/universe.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 2
+---
+
+# Universe
+
+The universe is where modules are displayed in a way that students can follow a certain direction or track.
+The role of the editor is to create these universes.
+
+## Viewing the universe
+
+When viewing the universe there are a couple of shortcuts and inputs to remember.
+
+| Action         | Result              |
+|----------------|---------------------|
+| R              | Reset view          |
+| CTRL + F       | Search for exercise |
+| Scroll         | Zoom in and out     |
+| Click and drag | Look around         |
```

Comment:
Dit voelt als iets om maandag ff qua style te bespreken.

Dit is letterlijk wat je ook terug ziet in de UI gwn copy/paste text. Ik weet niet of we dit willen doen, kan out of sync raken

Reply:
Zelfde voor het herhalen van de Universe clone warning,

Reply:
Hmm, ja voor nu ff laten staan en het er maandag ff over hebben?